docs/user-guide/builtins.md

```diff
@@ -116,6 +116,74 @@ Such named modules will always then be available under the
     Please see the documentation (linked above) about restrictions and gotchas
     when configuring how JavaScript modules are made available to PyScript.
 
+### `pyscript.fetch`
+
+A common task is to `fetch` data from the web via HTTP requests. The
+`pyscript.fetch` function provides a uniform way to achieve this in both
+Pyodide and MicroPython. It is closely modelled on the
+[Fetch API](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) found
+in browsers with some important Pythonic differences.
+
+The simple use case is to pass in a URL and `await` the response. Remember, in
+order to use `await` you must have the `async` attribute in the `script` tag
+that references your code.
+
+```python title="A simple HTTP GET with pyscript.fetch"
+from pyscript import fetch
+
+
+response = await fetch("https://example.com")
```

Comment:
this is not going to work ... you need to explicitly state what do you want from it:

  * `await fetch(url).text()`
  * `await fetch(url).json()`
  * `await fetch(url).bytearray()`
  * `await fetch(url).arrayBuffer()`
  * `await fetch(url).blob()`

These are the options and how the fetch works.

Reply:
Ahhh... OK. I'll update.